opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp

```diff
@@ -31,6 +30,8 @@ class EclipseGrid;
 class Deck;
 class DeckKeyword;
 namespace Fieldprops {
+class TranCalculator;
+using TranMap = std::unordered_map<std::string, TranCalculator>;
```

Comment:
Hm, this raises an interesting question.  We now have to declarations of `TranMap`&mdash;one here and one in [`TranCalculator.hpp`](https://github.com/OPM/opm-common/blob/a47716e97c36f119081d42983480df440b15538b/opm/input/eclipse/EclipseState/Grid/TranCalculator.hpp#L120).  The two declarations are equal right now, but how do we ensure that they remain equal?

Reply:
>>  how do we ensure that [the two declarations of `TranMap`] remain equal?
>
> We don't really. My personal preference would be to drop the typedef,

Yeah, I think that'd be the best option here really.  As far as I can tell, there's nothing in the current master sources that **depends** on the `TranMap` name.  We can't know anything about out-of-tree uses, but we generally don't really care about those anyway...

Reply:
done.